<launch>
	<node name="left_usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
		<param name="video_device" value="/dev/video0" />
		<param name="image_width" value="640" />
		<param name="image_height" value="480" />
		<param name="pixel_format" value="yuyv" />
		<param name="camera_frame_id" value="usb_cam_1" />
		<param name="io_method" value="mmap"/>
		<param name="framerate" value="40"/>
		<remap from="/left_usb_cam/image_raw" to="left_raw" />
	</node>

	<node name="right_usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
		<param name="video_device" value="/dev/video1" />
		<param name="image_width" value="640" />
		<param name="image_height" value="480" />
		<param name="pixel_format" value="yuyv" />
		<param name="camera_frame_id" value="usb_cam_2" />
		<param name="io_method" value="mmap"/>
		<param name="framerate" value="40"/>
		<remap from="/right_usb_cam/image_raw" to="right_raw" />
	</node>
</launch>
